client/src/selectors/index.js

```diff
@@ -18,7 +18,8 @@ import type {NoticesOrdering} from '../components/Notices/NoticeList'
 import type {NoticeDetailProps} from '../components/Notices/NoticeDetail'
 
 import type {CompanyDetailProps} from '../dataWrappers/CompanyDetailWrapper'
-import type {State, MapOptions, CompanyEntity, MapBounds, Company, NewEntityDetail} from '../state'
+import type {State, MapOptions, CompanyEntity, MapBounds, Company, NewEntityDetail, Notice} from '../state'
```

Comment:
There are still differences between mine and this (e.g. mine formats this line as multi-line).
Maybe I have nonstandard config - please ask someone else to review this too.